### ISSUE TITLE
Wrap into one module/package

### ISSUE THREAD

**Issue by jtoumey** [etoumey/bodyStats#42]
Would you consider wrapping the whole application such that it could be executed from one script? And it could prompt you to fetch more reports, or you could decide not to? @etoumey @npbassett

**Comment by etoumey** [comment]
Yeah, I have a shell script to do it. But since I'm on a different OS I haven't committed

**Comment by jtoumey** [comment]
Great. I'll refactor into a python module version .

**Comment by jtoumey** [comment]
* [ ] Modify functions such that they may be called individually from the command line or imported as a module 
* [ ] Write a `bodyStats.py` script which imports and calls the functions in a logical manner
